usagov/usagov-2021
# USAGOV-1979: add script to create domain services for www and cms with appropriate plans

## Jira Task


https://cm-jira.usa.gov/browse/USAGOV-1979

## Description

Adds a script to create the "external domain" services for www and cms, with space-appropriate values. Also removes a few commented lines showing the old values from the deploy-waf script. 

## Type of Changes

  - Add screenshot showing what it should look like
- [x] Other - deployment hints

## Testing Instructions



### Change Requirements



### Validation Steps

 - Read through the script and verify it makes sense to you.
 - Run the script with the --dryrun flag and verify it doesn't complain on your machine.

## Security Review




## Reviewer Reminders

- Reviewed code changes
- Reviewed functionality
- Security review complete or not required

## Post PR Approval Instructions

Follow these steps as soon as you merge the new changes.

1. Go to the [USAGov Circle CI project](https://app.circleci.com/pipelines/github/usagov/usagov-2021).
2. Find the commit of this pull request.
3. Build and deploy the changes.
4. Update the Jira ticket by changing the ticket status to `Review in Test` and add a comment. State whether the change is already visible on [cms-dev.usa.gov](http://cms-dev.usa.gov/) and [beta-dev.usa.gov](http://beta-dev.usa.gov/), or if the deployment is still in process.
